python/pyarrow/scalar.pxi

```diff
@@ -610,12 +609,10 @@ cdef class StructScalar(Scalar, collections.abc.Mapping):
     def __getitem__(self, key):
         """
         Return the child value for the given field.
-
```

Comment:
this whitespace was removed by accident? (in any case rst requires an empty line here) Same fore a few lines below

Reply:
Yes, when I reverted.